test/index.test.js

```diff
@@ -25,7 +25,7 @@ describe('#fetch', function () {
     it('should convert a HAR object to a HTTP request object', async function () {
       const request = constructRequest(harExamples.full);
 
-      expect(request.url).to.equal('https://httpbin.org/post?key=value?foo=bar&foo=baz&baz=abc');
```

Comment:
Did we have a test here for the broken case?

Reply:
we did and the broken case had the broken case ass its assertion